django_cloud_deploy/skeleton/templates/admin_template/admin.py-tpl

```diff
@@ -1 +1,69 @@
-#Custom Admin Code
\ No newline at end of file
+import os
```

Comment:
We should use 4 spaces indentation for this file to be consistent with other Django files.